lib/testrail.js

```diff
@@ -21,10 +21,31 @@ class TestRail {
 	}
 
 	async getCases(projectId, suiteId) {
+		let fullArr = [];
 		try {
 			const res = await this.axios.get(`get_cases/${projectId}&suite_id=${suiteId}`);
 			output.log(`getCases: SUCCESS - the response data is ${JSON.stringify(res.data)}`);
-			return res.data.cases;
+			let isLimit = false
+			let offset = 250;
+			fullArr = res.data.cases;
+			//limit logic works only if we suspect that api call send back limited array === 250 cases
+			if(res.data.cases.length === offset){
```

Comment:
no, this is bad. We cannot simply relie on this logic to tell if the user is on a specific TR version or not. For example, what if I'm on `7.0.1` which is what I'm currently on and I have strictly 250 cases. That'd break for me, so we need another way to figure this one out.

Reply:
what is better have separated flag, or make just one extra call if cases_id === 250, and don't care about versions at all. It's very rare case and set separated flag for that one case it's overhead. You config right now has a lot of flags and should be more easy to use, for example "closeTestRun" should be true by default. Who need opened (not closed) runs after test failed/passed ? @vladbalanescu @PeterNgTr